nym-wallet/src/components/TestNode/PrintResults.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import { Card, CardContent, CardHeader, Dialog, Divider } from '@mui/material';
+import { NymLogo } from '@nymproject/react/logo/NymLogo';
+import { ResultsCardDetail } from './ResultsCard';
+import { sleep } from 'src/utils/sleep';
+
+export const PrintResults = ({
+  packetsSent,
+  packetsReceived,
+  score,
+  mixnodeId,
+  mixnodeName,
+  date = '',
+  OnPrintRequestComplete,
+}: {
+  packetsSent: number;
+  packetsReceived: number;
+  score: number;
+  mixnodeId: string;
+  mixnodeName: string;
+  date?: string;
+  OnPrintRequestComplete: () => void;
+}) => {
+  const asyncPrint = async () => {
+    await sleep(250);
+    window.print();
+  };
```

Comment:
Just out of curiosity, why is some `sleep` needed?

Reply:
I was getting tired :laughing: JK.

I did it this way because window.print seems to run before the new component is rendered on the page. This causes it to print before the component (I want to print) is visible.

Hopefully that makes sense :thinking: